templates/compose/windmill.yaml

```diff
@@ -89,26 +90,26 @@ services:
       - worker-dependency-cache:/tmp/windmill/cache
       - worker-logs:/tmp/windmill/logs
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8000/api/version || exit 1"]
+      test: ["CMD-SHELL", "exit 0"]
       interval: 30s
       timeout: 10s
       retries: 3
 
   windmill-worker-native:
     image: ghcr.io/windmill-labs/windmill:main
     environment:
-      - DATABASE_URL=postgres://postgres:$SERVICE_PASSWORD_POSTGRES@db/windmill
-      - MODE=${MODE:-worker}
-      - WORKER_GROUP=${WORKER_GROUP:-native}
-      - NUM_WORKERS=${NUM_WORKERS:-8}
```

Comment:
Why were all these variables removed?

Reply:
I think we may be talking cross-purposes. Your change looks similar to the following:

```diff
- - NUM_WORKERS=${NUM_WORKERS:-8}
+ - NUM_WORKERS=8
```

That means you have _removed_ my ability to specify `NUM_WORKERS` as an environment variable to specify how many (native) workers I want.